src/DGtal/geometry/volumes/distance/ExactPredicateLpPowerSeparableMetric.ih

```diff
@@ -311,7 +311,7 @@ DGtal::ExactPredicateLpPowerSeparableMetric<T,2,P>::binarySearchHidden(const Abs
 								       const Abscissa &/*upper*/) const
 {   
   ASSERT(false && "Not Necessary for l_2");
-  
+  return 0;
```

Comment:
Should we not better thrown an exception or simply not define this function (only the header) ?